src/ngraph/runtime/cpu/builder/sigmoid.cpp

```diff
@@ -109,8 +114,132 @@ namespace ngraph
                 functors.emplace_back(functor);
             }
 
+            template <typename ElementType>
+            static Eigen::TensorMap<Eigen::Tensor<ElementType, 1, Eigen::RowMajor>>
+                wrap_into_tensor_map(void* data, size_t tensor_size)
+            {
+                Eigen::array<Eigen::Index, 1> dims;
+                dims[0] = tensor_size;
+
+                Eigen::TensorMap<Eigen::Tensor<ElementType, 1, Eigen::RowMajor>> out(
+                    static_cast<ElementType*>(data), dims);
+                return out;
+            }
+
+            template <>
+            void Builder::BUILDER_DECL(ngraph::op::SigmoidMultiply)
+            {
+                auto& functors = external_function->get_functors();
+                auto& tensor_data = external_function->get_tensor_data();
+
+                auto& arg0_tensor = tensor_data[args[0].get_name()];
+                auto& arg1_tensor = tensor_data[args[1].get_name()];
+                auto& out_tensor = tensor_data[out[0].get_name()];
+                auto tensor_size = shape_size(args[0].get_shape());
+
+                auto sigmoid_mul = static_cast<const ngraph::op::SigmoidMultiply*>(node);
+                function<void(CPURuntimeContext*)> functor;
+
+                const size_t index =
+                    static_cast<size_t>(sigmoid_mul->get_input_func_type(0)) *
+                        static_cast<size_t>(ngraph::op::SigmoidMultiply::FunctionType::NumTypes) +
+                    static_cast<size_t>(sigmoid_mul->get_input_func_type(1));
+
+                switch (index)
+                {
+                case 0 /*Logistic|Logistic*/:
+                    functor = [&, tensor_size](CPURuntimeContext* ctx) {
```

Comment:
Does this op enforce an element type of float? Otherwise you'll want to disallow other types

Reply:
yes... we only fuse for floats.

Reply:
Cool. So the framework can't create instantiate this op all by itself, is that right?